tests/test_interrupted_sync.py

```diff
@@ -0,0 +1,282 @@
+"""
+Test tap successfully resumes after interrupted sync without missing any records
+"""
+from datetime import datetime as dt
+
+from tap_tester import menagerie, connections, runner, LOGGER
+from base import BaseTapTest
+
+
+class InterruptedSyncTest(BaseTapTest):
+    """Test tap sets a bookmark and respects it for the next sync of a stream"""
+    @staticmethod
+    def name():
+        return "tap_tester_shopify_int_sync_test"
+
+    # function for verifying the date format
+    def is_expected_date_format(self, date):
+        try:
+            # parse date
+            dt.strptime(date, "%Y-%m-%dT%H:%M:%S.%fZ")
+        except ValueError:
+            # return False if date is in not expected format
+            return False
+        # return True in case of no error
+        return True
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.start_date = '2022-03-01T00:00:00Z'
+
+    def test_run(self):
+
+        conn_id = self.create_connection(original_properties=False, original_credentials=False)
+
+        interrupt_test_streams = {'collects',
+                                  'customers',
+                                  'events',
+                                  'metafields',
+                                  'orders',
+                                  'products',
+                                  'transactions'}
```

Comment:
A couple of the other tests use `expected_streams()` for this set. Is there a reason we are breaking that pattern?

Reply:
Intent was to simplify the test be limiting streams.  Will further reduce stream count and use standard naming.